### DISCUSSION TITLE
As a user, in a modal I should be able unset focus from input fields

### DISCUSSION THREAD

**Discussion by shashank1010** [GeekyAnts/NativeBase#5176] [Feature request]
### Description

When you open the modal for this example
https://snack.expo.dev/@shashank1010/modal-checkbox-click-issue

You would notice that the first field has focus.
In this example, I simulated a vertical space with a height of 500px and then there is a checkbox.
When, the first field has focus and you scroll down to click on the checkbox, modal body scrolls up to the first field.

Also, in this case, after scrolling down, if I click on the modal body itself, even then it scrolls up to the first field.
The example in the doc works beautifully, however, only when there is no scroll.

### Problem Statement

In a modal that has input fields, scrolling and then clicking outside a field scrolls the modal to the last selected field.

### Proposed Solution or API

...
<FocusScope
              contain={visible && props.containScope}
              .
              .
              .
              .
> ... </FocusScope>

### Alternatives

I've tried all possible hacky solutions to resolve this issue, but the field just keeps retaining focus.

### Additional Information

https://snack.expo.dev/@shashank1010/modal-checkbox-click-issue

**Comment by Krithikj20** [comment]
Hi @shashank1010, Thank you for reporting this issue. We will try to fix it at the earliest.

**Comment by shashank1010** [comment]
@Viraj-10 to your comment on issue before making this into a discussion.

I believe the link to the snack did reproduce the issue reliably.
In case you thought that the white space is redundant, it was a shortcut to emulate some content/fields in between. I have since added garbage text make the modal scroll.

Imagine a popup window with several fields, such that the modal body scrolls.
Imagine a consent checkbox at the bottom.

when you click on the checkbox, the last selected field gets the focus modal body scrolls down to the focused field.
Imagine the frustration of the user who cannot select a required checkbox to proceed.

**Reply by Viraj-10** [reply]
Hi @shashank1010, This Discussion is converted into issues (https://github.com/GeekyAnts/NativeBase/discussions/5176) again and will be fixed in the next release. sorry for the inconvenience. you can also fix it for temporary by passing ` _web={{ focusable: false }}` to `Modal.Content`
